lametro/bills.py

```diff
@@ -94,9 +94,33 @@ def actions(self, matter_id) :
 
                 yield bill_action, votes
 
-    def scrape(self, window=28) :
```

Comment:
This change should be made in https://github.com/opencivicdata/scrapers-us-municipal not in this repo.

Reply:
It was: https://github.com/opencivicdata/scrapers-us-municipal/pull/240 You are seeing the git diff for my merge with master.